server/controllers/student.py

```diff
@@ -214,17 +187,12 @@ def group_remove(course, assign):
             group.remove(current_user, target)
         except BadRequest as e:
             flash(e.description, 'danger')
+    return redirect(url_for('.assignment', offering=assignment.course.offering, name=assignment))
```

Comment:
The group actions redirects: `name=assignment` should be `name=assignment.name` 

I don't offering is actually necessary - it would just be added as a query parameter (and isn't used by the route anyway)